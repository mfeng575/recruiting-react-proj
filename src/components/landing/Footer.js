import React from 'react';

const Footer = () => {
    return (
        <footer>
            <div >
                @copyright PGA 2019
            </div>
        </footer>
    );
};

export default Footer;
