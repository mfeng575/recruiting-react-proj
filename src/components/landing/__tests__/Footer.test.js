import React from 'react';
import {configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
import toJson from 'enzyme-to-json';
import Footer from '../Footer';


describe('Footer.test.js', () => {

    it('renders without crashing', () => {
        
        const wrapper = shallow(<Footer />);

        expect(wrapper).toHaveLength(1);

        expect(wrapper.find('footer')).toHaveLength(1);
        
        const tree = toJson(wrapper);
        expect(tree).toMatchSnapshot();
        
    });
});
