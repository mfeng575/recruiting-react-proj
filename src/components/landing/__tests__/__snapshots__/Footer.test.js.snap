// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.test.js renders without crashing 1`] = `
<footer>
  <div>
    @copyright PGA 2019
  </div>
</footer>
`;
